pkg/eventbus/event_bus.go

```diff
@@ -8,18 +8,30 @@ import (
 	"github.com/kubeedge/kubeedge/beehive/pkg/common/log"
 	"github.com/kubeedge/kubeedge/beehive/pkg/core"
 	"github.com/kubeedge/kubeedge/beehive/pkg/core/context"
-
 	"github.com/kubeedge/kubeedge/pkg/eventbus/common/util"
 	mqttBus "github.com/kubeedge/kubeedge/pkg/eventbus/mqtt"
 )
 
+var mqttServer *mqttBus.MqttServer
+
+const (
```

Comment:
maybe it will look better to change the const name of doubleMqttMode to  bothMqttMode?